README.md

```diff
@@ -1,23 +1,63 @@
-# VQM
-## Video Quality Metrics
-This repository contains the Video Quality Metric (VQM) software developed by ITS. A full description of each of the files provided here can be found <a href="https://www.its.bldrdoc.gov/resources/video-quality-research/guides-and-tutorials/description-of-vqm-tools.aspx" target="_blank"> here on the ITS Web Site</a>. The VQM tools are available free of charge for any commercial or non-commercial use, in accordance with the terms described in the [LICENSE](https://github.com/NTIA/vqm/blob/master/LICENSE.md).
+# Video Quality Metric (VQM) Archival Repository
 
-For each software package, there are three download options: the MATLAB® source, a 32-bit compiled version, and a 64-bit compiled version. In each case, the relevant documentation is provided. 
+This repository provides an archival record of video quality metric (VQM) research performed by ITS between 1989 and 2011. 
+This research focused on full reference (FR) and reduced reference (RR) methods, which compare a processed video sequence with an original, high quality video sequence. 
+Thus, quality estimation for a processed video sequence requires access to the original, high quality video sequence. 
+
+This software has not been maintained since 2013 and is provided for archival and research purposes only. 
+ITS does not have the funding to perform troubleshooting. 
+All souce code was developed for MATLAB® R2013b (8.2). Future versions of MATLAB may cause the code not to run.
```

Comment:
Suggest: All [souce] source code was developed for MATLAB® R2013b (8.2). The code may not to perform as expected in other versions.